build.sbt

```diff
@@ -1,26 +1,34 @@
+import sbt.Def
 import scoverage.ScoverageKeys
-import uk.gov.hmrc.DefaultBuildSettings.{defaultSettings, scalaSettings, targetJvm}
+import uk.gov.hmrc.DefaultBuildSettings.{defaultSettings, scalaSettings}
 
 val appName = "tamc-frontend"
 
-lazy val excludedPackages = Seq(
-  "<empty>",
-  "app.*",
-  "config.*",
-  "testOnlyDoNotUseInAppConf.*",
-  "views.*",
-  "uk.gov.hmrc.*",
-  "prod.*",
-  "forms.*",
-  "connectors.ApplicationAuditConnector",
-  "connectors.ApplicationAuthConnector",
-  "services.CachingService",
-  "metrics.Metrics",
-  "utils.WSHttp",
-  "events"
+ThisBuild / scalaVersion := "2.13.12"
+ThisBuild / majorVersion := 7
+ThisBuild / scalacOptions ++= Seq(
+  "-feature",
+  //"-Xfatal-warnings" FIXME switch to HMRCStandardPage template instead of deprecated HmrcLayout
```

Comment:
This line is intentionally commented out, fixing the layout would increase the complexity of this ticket (and there are some comments that need to be addressed before updating the layout)